operator/config/samples/lifecycle_v1alpha1_keptntaskdefinition_container.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: lifecycle.keptn.sh/v1alpha1
+kind: KeptnTaskDefinition
+metadata:
+  name: keptntaskdefinition-sample-container
+spec:
+  container:
+    name: testy-test
+    image: busybox:1.36.0
+    command:
+      - 'sh'
+      - '-c'
+      - 'sleep 30'
```

Comment:
I would remove this from the example - in alpha1 we didn't support it. You have the conversion webhook, but that requires to deploy a recent operator version. 
I would instead add an example for v1alpha3